onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -14,6 +14,7 @@
 #include "core/framework/session_state.h"
 #include "core/framework/op_kernel_context_internal.h"
 #include "core/framework/utils.h"
+#include "nlohmann/json.hpp"
```

Comment:
Just remembered that this will increase the binary size unconditionally. How much is the increase (please use the rel branch to calculate before and after)? If the increase is significant, may be for this release we can fix the issue without using this lib.

Reply:
the lib is already introduced by session.lib, the dependency we add here actual reuse the binary no?

Reply:
The inclusion is protected by a macro: https://github.com/microsoft/onnxruntime/blob/c716b56f26215788e35aca04644b51a511a4c2bb/onnxruntime/core/session/inference_session_utils.h#L8-L16

Reply:
May be we should exclude it from minimal build as well.

Reply:
We shouldn't disable the whole profiling feature for minimal builds.

Reply:
no - we just skip type and shape details.